app/src/composables/use-permissions.ts

```diff
@@ -54,9 +54,9 @@ export function usePermissions(collection: Ref<string>, item: Ref<any>, isNew: R
 
 		if (!permissions) return fields;
 
-		if (permissions?.fields?.includes('*') === false) {
+		if (permissions.fields?.includes('*') === false) {
 			fields = fields.map((field: Field) => {
-				if (permissions.fields.includes(field.field) === false) {
+				if (permissions.fields?.includes(field.field) === false) {
```

Comment:
Hey @paescuj,
I think we don't need `?` because condition on 57 already handles that, right?
`undefined != false`

Reply:
@joselcvarela TypeScript still yells at you, seeing the usage is within a different function scope, which technically means that the value of the referenced object could've changed between the if on 57 and the execution of this callback function 👍🏻